Tasks/Common/packaging-common/nuget/NuGetExeXmlHelper.ts

```diff
@@ -26,24 +27,41 @@ export class NuGetExeXmlHelper implements INuGetXmlHelper {
         nugetTool.arg("-ConfigFile");
         nugetTool.arg(this._nugetConfigPath);
 
+        let updatePassword = false;
+        const tempPassword = '***redacted***';
```

Comment:
I'd prefer this to be (or include) something like a GUID, which is more-or-less guaranteed not to appear in the user's file. Doesn't have to be newly-generated every time. If you do use a GUID, leave a comment in the source saying it's there because it's extraordinarily unlikely to appear in the customer's file.